gnat2goto/driver/driver.adb

```diff
@@ -97,6 +100,16 @@ package body Driver is
          end loop;
       end;
 
+      if not Add_Start then
+         Put_Line (Create (SymbolTable2Json (Global_Symbol_Table)).Write);
+         return;
+      end if;
+
+      Program_Expr := New_Irep (I_Symbol_Expr);
```

Comment:
If these are now only used here and below, it would be usual to declare them here using a `declare ... begin ... end` block rather than declare them above then only start using them here.